### PR TITLE
Add forms

### PR DESCRIPTION
**Pull request by eudondian**
### **Add forms**

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-  root 'users#index'
+    root 'users#index'
   resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show]
+    resources :posts, only: [:index, :new, :show] do
+      resources :comments, only: [:new, :create] 
+      resources :likes, only: [:create] 
+    end
   end
+  # post '/posts', to: 'posts#create'
```
Comment (t-yanick) [inline config/routes.rb]: - [ ] Great job working on this milestone. In to maintain our repo in a professional way, we should always deleted all commented lines of code. We can always re-visit old codes with git history

File: app/controllers/comments_controller.rb [hunk 2]
```diff
@@ -1,17 +1,28 @@
...
+    respond_to do |format|
+      format.html { render :new, locals: { comment: } }
+    end
+  end
 
-  def comment_params
-    params.require(:comment).permit(:text)
+  def create
+    @current_user = current_user
+    comment = Comment.new(params.require(:comment).permit(:post_id, :text).merge(author_id: @current_user.id))
```
Comment (t-yanick) [inline app/controllers/comments_controller.rb:15]: - [ OPTIONAL] 
You could add a `private` params method to implement strong parameters. You can visit this [link](`https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md#:~:text=Pay%20special%20attention%20to%20the%20strong%20parameters%20description%20in%20Rails%20Guides`) to read more about strong parameters.

**Example**
```ruby
  private 
  def comment_params
    params.require(:comment).permit(:text)
  end
```

**Review by t-yanick — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED

Hi @eudondian , 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  ApplicationController contains `#current_user` method
:heavy_check_mark: The posts and comments controller  have new and create actions
:heavy_check_mark: There is  a pair of routes in the config/routes.rb file for creating posts and comments.
:heavy_check_mark:  Use of correct gitflow

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by codepantha — APPROVED** [review]
# STATUS: APPROVED 🟢 ✔️ 

## Hi @eudondian 👋, 

Great job implementing the required changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights 🎯 
✔️ Descriptive PR
✔️ `Post` and `Comment` controllers have `new` and `create` actions 📌 
✔️ `ApplicationController` contains `#current_user` method 📌 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**